playbooks/common/openshift-master/config.yml

```diff
@@ -337,12 +361,54 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
-  hosts: oo_masters_to_config
+- name: Configure first master instance
```

Comment:
This is actually on my todo list to clean up I just couldn't figure out any other way to do it to get going.

If I'm following, openshift_version isn't used as an inventory variable, it's only set in the playbooks/roles and it meant to be the calculated variable we actually use, after taking release / pkg_version / image_tag into account.

openshift_release would be the misleadingly similarly named inventory variable. 

So how can we just use one play here? It looked to me like I must (a) calculate openshift_version based on all other inputs for first master, (b) install first master so it has a openshift.common.version fact, and then (c) install all other masters. I can't think of a way to do this without separation.

I also just noticed the subsequent masters are not properly protecting their installed versions, i.e. if first master somehow got a newer version, a re-run of config playbook would upgrade the subsequent masters. Adding that to TODO as well.

I'm thinking I need another play in between these to sort out the openshift_version fact for non-first masters, but still requires separate copied plays.